RecoDataProducts/inc/STMWaveform.hh

```diff
@@ -0,0 +1,51 @@
+#ifndef RecoDataProducts_STMWaveform_hh
```

Comment:
Is there a potential name clash with the raw STM waveform 'digi' object?

Reply:
This is intended to be the raw STM waveform 'digi' object. Should I add "Digi" to the class name?

Reply:
I think having 'Digi' in the name would be helpful, but its up to you.

Reply:
I agree. Done in commit 3be8644